app/lib/feed_manager.rb

```diff
@@ -27,33 +27,41 @@ def filter?(timeline_type, status, receiver_id)
   end
 
   def push(timeline_type, account, status)
```

Comment:
Considering that we only ever use `push(:home)` (as far as I can tell....) and we're adding a push_to_list *anyway*, we should probably make a `push_to_home` method, and then either get rid of `push` or make it private and get rid of any code duplication between it and `push_to_list`